app/config.py

```diff
@@ -72,35 +89,17 @@ def _zone_from_USB(self):
                 self.zone = str(i)
                 return
 
-    def _get_team_specifics(self):
+    def _get_team_image(self):
         """Find information set on each brain about the team
 
         Only makes sense to run this if we are on a brain
         Teamname is set per brain before shipping and allows unique graphics
         for ID'ing teams in the arena.
-
-        Pick a start image in order of preference :
-            1) We have a team corner image on the USB
-            2) The team have uploaded their own image to the robot
-            3) We have a generic corner image on the USB
-            4) The game image
         """
+        teamname_jpg = 'none'
         if self.teamname_file.exists():
-            teamname_jpg = self.teamname_file.read_text().replace('\n', '') + '.jpg'
-        else:
-            teamname_jpg = 'none'
-
-        start_img_path = self.arena_usb_path / teamname_jpg
-        if not start_img_path.exists():
-            start_img_path = Path('usercode/editable/team_logo.jpg')
-        if not start_img_path.exists():
-            start_img_path = self.arena_usb_path / 'Corner.jpg'
-        if not start_img_path.exists():
-            start_img_path = Path('/home/pi/game_logo.jpg')
-
-        if start_img_path.exists():
-            displayed_img_path = Path('static/image.jpg')
-            displayed_img_path.write_bytes(start_img_path.read_bytes())
-
+            teamname_jpg = self.teamname_file.read_text().strip() + '.jpg'
+        
+        self.camera_image_files[1] = Path(teamname_jpg) 
```

Comment:
To a person reading only this function `1` is a bit of a magic number. Maybe a comment here to explain where this came from would be useful?